src/backend/src/utils/utils.ts

```diff
@@ -112,3 +112,18 @@ export const getCurrentUser = async (res: Response): Promise<User> => {
   if (!user) throw new NotFoundException('User', userId);
   return user;
 };
+
+export const countWords = (str: string): number => {
+  const words = str.split(' ');
+  let wordCount = 0;
+  words.forEach((word) => {
+    if (word.trim() !== '') {
+      wordCount++;
+    }
+  });
+  return wordCount;
```

Comment:
https://stackoverflow.com/questions/18679576/counting-words-in-string#:~:text=function%20WordCount(str)%20%7B%20var,space%20%3D%202%20words%20%7D%20console.

maybe some simpler stuff here idk